setup.py

```diff
@@ -47,23 +52,30 @@
     ],
     package_dir={"": "src"},
     packages=setuptools.find_packages(where="src"),
-    python_requires=">=3.6",
+    include_package_data=True, 
+    package_data={
+        "": ["*.sh", "*.crt", "*.key"], 
+    },
+    python_requires=">=3.8",
     install_requires=[
```

Comment:
Dependent packages
- I think most users would not need globus. The packages related to globus can be optional.
- Do we need `torchvision` and `matplotlib` as required packages?
- `tensorboard` is removed. Do we not support it anymore?

Reply:
1. It can be optional, but as all globus sdks are very light-weight, do you think it is really necessary to mark them optional.
2. Great catch! `torchvision` is not required and I will move it to `examples`. `matplotlib` is required as it is used in some misc functions within `appfl.misc`
3. I will add tensorboard back, some codes still support tensorboard